base/libgit2/blob.jl

```diff
@@ -4,15 +4,28 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+"""
+    rawcontent(blob::GitBlob) -> Array
+
+Fetch the *raw* contents of the [`GitBlob`](@ref) `blob`. This is an
+`Array` containing the contents of the blob, which may be binary or may be Unicode.
```

Comment:
"may or may not be valid Unicode"? I find "may be binary or may be Unicode" a little confusing since it's always binary; it doesn't return `Char`s.